### PR TITLE
Fix type annotation

### PR DESCRIPTION
**Pull request by zhiqwang**

### CODE REVIEW

File: onnx2torch/utils/safe_shape_inference.py [hunk 1]
```diff
@@ -12,7 +12,11 @@ def _is_big_model(model: ModelProto) -> bool:
     return model.ByteSize() / (1024 * 1024 * 1024) > 2.0
 
 
-def _shape_inference_by_model_path(model_path: Union[Path, str], output_path: [Path, str], **kwargs) -> ModelProto:
+def _shape_inference_by_model_path(  # pylint: disable=missing-function-docstring
+    model_path: Union[PosixPath, str],
+    output_path: Union[PosixPath, str],
+    **kwargs,
+) -> ModelProto:
```
Comment (ivkalgin) [inline onnx2torch/utils/safe_shape_inference.py:19]: Why you replace Path by PosixPath? What is the reason?

Comment (zhiqwang) [reply onnx2torch/utils/safe_shape_inference.py:19]: @ivkalgin Sorry, I've been misrepresenting `PosixPath` and `Path`, I'll change it back here

File: onnx2torch/utils/safe_shape_inference.py [hunk 2]
```diff
@@ -12,7 +12,11 @@ def _is_big_model(model: ModelProto) -> bool:
     return model.ByteSize() / (1024 * 1024 * 1024) > 2.0
 
 
-def _shape_inference_by_model_path(model_path: Union[Path, str], output_path: [Path, str], **kwargs) -> ModelProto:
+def _shape_inference_by_model_path(  # pylint: disable=missing-function-docstring
```
Comment (ivkalgin) [inline onnx2torch/utils/safe_shape_inference.py:15]: pylint annotation is not necessary and should be removed, because function is private '_'.

Comment (zhiqwang) [reply onnx2torch/utils/safe_shape_inference.py]: Done, thanks @ivkalgin !

**Comment by ivkalgin** [comment]
This is insignificant code style changes. Current code and your code corresponds to black code style.

**Comment by zhiqwang** [comment]
Just FYI @ivkalgin , there missed a `Union` at `output_path`

https://github.com/ENOT-AutoDL/onnx2torch/blob/a3f04ad6c0987752c8f338625b78e8952d25a9be/onnx2torch/utils/safe_shape_inference.py#L15

**Comment by senysenyseny16** [comment]
@zhiqwang oh, ty!

Interestingly, `pylint` and `black` miss this typo :(

**Pull request merged**